### ISSUE TITLE
Cannot duplicate the accuracy result of experiment in office-31 A-D

### ISSUE THREAD

**Issue by FLAWLESSJade** [tim-learn/ATDOC#3]
Hi~ Thanks for your codes， it helps me so much to understand your paper. 
Recently I use this code to run related experiments in the dataset: OFFICE-31 with the command：

**python demo_uda.py --pl atdoc_na --tar_par 0.2 --dset office --max_epoch 100 --s 0 --t 1 --gpu_id 0 --method DDC --output logs/uda/run1/**

However, the experiment result of office-31 in A to D cannot achieve 96% in your paper but got 89%， which made me so confused.
I have tried to revise the batch size to 16 or 28，but does not work.
 (Cause of my GPU memory limitation, I cannot set the batch size to 32 or 36 )
Can you share more detail of the hyper-parameter for this dataset training, I will be very grateful.

**Comment by FLAWLESSJade** [comment]
Ok， I solved this question！
 I test many parameters in this dataset training, which includes turning up the ratio for loss function and revising the batch size from 16 24, and 28, etc. whatever, it did not work, in the end.
Finally, my friend lent her GPU to me, and I use the default setting with batch size：36. 
It gets work. 
GPU type: RTX3090 24G 
THAT IS THE POWER OF MONEY!
